src/cart/src/cart/crt_ctl.h

```diff
@@ -45,6 +45,7 @@
 /* crt uri lookup cache info */
 struct crt_uri_cache {
 	struct crt_grp_cache	*grp_cache;
+	size_t			 max_count;
```

Comment:
uint_32t would make more sense, it's not a size.

Reply:
Done